src/averageAges.js

```diff
@@ -56,6 +87,23 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const haveMothers = people
+    .filter(person => people.some(child => child.mother === person.name));
+  const DiffMotherChild = [];
+
+  haveMothers.map(mother => {
```

Comment:
Why map? The map can’t change something outside, but you change DiffMotherChild array